tests/cypress/integration/reportRepair/repairProblemBestDescription.spec.js

```diff
@@ -358,7 +354,7 @@ describe('repairProblemBestDescription', () => {
       testSinkOptions();
     });
 
-    context('Electrics, including extractor fan and pull cords', () => {
+    context.only('Electrics, including extractor fan and pull cords', () => {
```

Comment:
I think this `.only` would cause only this context to run in the tests. Should we remove this?

Reply:
Yes, accidentally left this in. Removed now.